enaml/scintilla/scintilla.py

```diff
@@ -104,6 +112,12 @@ class Scintilla(Control):
     on this widget. All styling is supplied via the 'theme' attribute.
 
     """
+    #: Enable autocompletion
+    autocomplete = d_(Enum('none', 'all', 'document', 'apis'))
+
+    #: Autocompletion values
+    autocompletions = d_(List(str))
```

Comment:
I wish there was a way to hook an autocompletion library (such as jedi) in Scintilla... but it does not appear possible.

Reply:
Never heard of jedi until now. IPython also has a [completer](https://github.com/ipython/ipython/blob/master/IPython/core/completer.py) that I've used.  Looks like either can be used. I'll see if I can add a examples.